bitnami/zookeeper/templates/_helpers.tpl

```diff
@@ -107,7 +170,16 @@ Return the name of the secret containing the Keystore and truststore password
 {{- end -}}
 
 {{/*
-Return the secret containing Zookeper client TLS certificates
+Return true if a TLS secret should be created for ZooKeeper client
+*/}}
+{{- define "zookeeper.client.createTlsSecret" -}}
+{{- if and .Values.tls.client.enabled .Values.tls.client.autoGenerated (not .Values.tls.client.existingSecret) }}
```

Comment:
Just to double check, looks like this old function used `tls.quorum` instead of `tls.client` is that change expected? Because I can see we are maintaining both objets

Reply:
I think the GitHub UI confused you. There are two named templates: "zookeeper.quorum.createTlsSecret" and "zookeeper.client.createTlsSecret". The former uses `tls.quorum`, while the latter uses `tls.client`

Reply:
👍